modules/ROOT/pages/organization-management-in-bonita-bpm-studio.adoc

```diff
@@ -93,7 +93,7 @@ In order to log in to bonita UIs (Bonita User Application, Bonita Administration
 
 *_In Community_*, only provided profiles are supported (User and Adminstrator). When deploying an organization from the Studio, each user is automatically mapped to all those profiles.
 
-*_In Teamwork edition_*, 3 provided profiles (User, Process Manager and Adminstrator) can be edited in the Studio using an XML editor (Go to Organization > Profiles > Open). For development purposes, all users mapped to role _member_ (cf _ACME_ organization) will benefit from all profiles in the Portal to let you log in with any of those users and test your processes.
+*_In Teamwork edition_*, 3 provided profiles (User, Process Manager and Adminstrator) can be edited in the Studio using an XML editor (Go to Organization > Profiles > Open). For development purposes, all users mapped to role _member_ (cf _ACME_ organization) will benefit from all profiles in the Bonita Applications to let you log in with any of those users and test your processes.
```

Comment:
User, Process Manager and Adminstrator) => User, Process Manager and Administrator)

Reply:
Done, I also changed other places in the same file where Administrator was misstyped.